custom/background-effects/README.md

```diff
@@ -0,0 +1,28 @@
+# Background Effects
+
+### Live example
+
+**[See it in action here ➡️](https://custom-background-effects.vercel.app)**
+
+---
+
+## What does this demo do?
+
+- Use [updateInputSettings](https://docs.daily.co/reference/daily-js/instance-methods/update-input-settings) to add effects to your background
+- Implements a custom `<Tray />` that adds Background effects option to the tray.
+
+Please note: this demo is not currently mobile optimised
+
+### Getting started
+
+```
+# set both DAILY_API_KEY and DAILY_DOMAIN
+mv env.example .env.local
+
+yarn
+yarn workspace @custom/background-effects dev
+```
+
```

Comment:
I noticed you're checking the domain setting too, so we should explicitly include those as options as well. For any of these demos, we want to say as much as possible in the README about how it works so the reader doesn't have to learn from digging through the code. :)

"This demo requires a Daily [room](https://docs.daily.co/reference/rest-api/rooms/config#enable_video_processing_ui) or [domain](https://dashboard.daily.co/rooms-settings) with the `enable_video_processing_ui` option turned on. This can be done via Daily's [REST API](https://docs.daily.co/reference/rest-api/rooms/create-room) or the [Daily dashboard](https://dashboard.daily.co/rooms/create)."